crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -126,6 +134,20 @@ async fn main() -> Result<(), Box<dyn Error>> {
             let decoded_keypair = Keypair::try_from_bytes(hex::decode(keypair)?.as_mut_slice())?;
             let keypair = identity::Keypair::from(decoded_keypair);
 
+            // Metrics
+            let mut metric_registry = Registry::default();
+            let metrics = metrics_port.map(|_| Metrics::new(&mut metric_registry));
+
+            let prometheus_task = metrics_port
+                .map(|port| {
+                    let address = format!("0.0.0.0:{port}")
```

Comment:
I don't like `0.0.0.0` as the default, I think both IP and port should be configurable. Moreover, there should be a vector of them so you can specify multiple. I fact you'll not need to parse, it'll be parsed by `clap` with (probably) helpful error message in case of errors.